test/test_autograd.py

```diff
@@ -6861,7 +6821,8 @@ def test_parameter_resize(self, device):
             m = torch.cat((asd, asd))
             m.sum().backward()
 
-    def test_sparse_ctor_getter_backward(self, device):
+    @dtypes(torch.double, torch.cdouble)
```

Comment:
Why double and cdouble (vs float and cfloat, for example)?

Reply:
It's because gradcheck for `torch.cfloat` and `torch.float` with default eps/tolerance will fail.